### DISCUSSION TITLE
Nx is rebuilding all our apps even when it doesn't affect them

### DISCUSSION THREAD

**Discussion by zackarydev** [nrwl/nx#19277] [General]
Ex:

adding `"@types/express": "~4.17.13"` causes all apps to be rebuilt! 

Or changing the README. The implicitDependencies property in Nx.json is deprecated. What is the latest way of fixing this issue?

**Comment by hakubo** [comment]
I think you can use nxignore for this: https://nx.dev/reference/nxignore
